docsite/js/script.js

```diff
@@ -1,7 +1,7 @@
  //modal close button
 (function(){
     //π.modalCloseButton = function(closingFunction){
-    //	return π.button('pi-modal-close-button', null, null, closingFunction);
+    // return π.button('pi-modal-close-button', null, null, closingFunction);
     //};
```

Comment:
This is all commented out? Can we drop it?

Reply:
It's actually not related to my change.
But I think it's better to remove those commented lines.

Reply:
I have removed it. thanks